src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -24,6 +24,10 @@
 
 namespace Microsoft.ML.Trainers
 {
+    /// <summary>
+    /// Train a matrix factorization model using LIBMF.
+    /// </summary>
+    /// <include file='doc.xml' path='doc/members/member[@name="MatrixFactorizationTrainer"]/*' />
```

Comment:
>   /// <include f [](start = 1, length = 17)

also, as FYI, you don't have to write the documentation int he doc, you can do it directly ehr ein code; up to you. 

The docs got created because we had components in multiple places: Runtime and CSHarpAPI

Reply:
Ok. I will move it alongside the code.

---
In reply to: [229413383](https://github.com/dotnet/machinelearning/pull/1450#discussion_r229413383) [](ancestors = 229413383)